packages/core/src/contenttype.ts

```diff
@@ -15,4 +15,6 @@ export const ContentType = {
   SVG: 'image/svg+xml',
   TEXT: 'text/plain',
   TYPESCRIPT: 'text/typescript',
-};
+} as const;
+
+export type ContentTypeString = (typeof ContentType)[keyof typeof ContentType];
```

Comment:
Why was this necessary?

I don't think we want to treat `ContentType` as exhaustive or restrictive.  It's just meant to be a convenient set of constants.